### PR TITLE
Bump to v2.6.0

### PR DESCRIPTION
**Pull request by alicethorne-ab**
Bumps the image tag to v2.6.0.

See this link for changelog: https://app-updates.agilebits.com/product_history/SCIM

### CODE REVIEW

**Comment by grellyd** [comment]
LGTM! ✅ 

I'll let Chas have a peek Monday before merging.

**Review by grellyd — APPROVED** [review]

**Comment by chasdl** [comment]
Perfect! Merging.

**Review by chasdl — APPROVED** [review]

**Pull request merged**
